frontend/src/components/category-pill.tsx

```diff
@@ -4,16 +4,24 @@ import { twMerge } from 'tailwind-merge';
 interface CategoryPillProps extends React.HTMLAttributes<HTMLSpanElement> {
   category: string;
   selected?: boolean;
+  disabled?: boolean;
 }
 
-export default function CategoryPill({ category, selected = false }: CategoryPillProps) {
+export default function CategoryPill({
+  category,
+  disabled,
+  selected = false,
+}: CategoryPillProps) {
+
+  const disabledColor: string = "opacity-50 bg-light-primary/10 dark:bg-dark-primary/10 dark:text-dark-primary/50 cursor-not-allowed"
   const [defaultColor, selectedColor]: [string, string] = getCategoryColors(category);
 
   return (
     <span
       className={twMerge(
-        'cursor-pointer rounded-3xl px-3 py-1 text-xs font-medium text-light-primary/80 dark:text-dark-primary/80',
-        selected ? selectedColor : defaultColor
+        'cursor-pointer rounded-3xl px-3 py-1 text-xs font-medium text-light-primary/80 dark:text-dark-primary/80 ',
+        selected ? selectedColor : defaultColor,
+        disabled ? disabledColor : null
       )}
```

Comment:
Hey @vivek6201, the complete PR is to solve this bug, it won't work this way, because it will take 2 colors at the same time, and tailwind-css gets confused which one to choose, it might seem it 2nd overrides 1st one, but not necessarily true